allay/plugin.py

```diff
@@ -0,0 +1,29 @@
+from beet import Context
+from beet.core.container import Drop
+from beet.core.file import TextFile
+from beet.contrib.messages import Message
+from beet.library.base import NamespaceFile
+from beet.library.data_pack import DataPack
+
+from tokenstream import InvalidSyntax
+
+from functools import cached_property
+
+from .parser import Parser
+
+class AllayMessage(TextFile, NamespaceFile):
+    scope = ("messages",)
+    extension = ".allay"
+
+    @cached_property
+    def parser(self) -> Parser:
+        return Parser()
```

Comment:
Per fizzy on discord, `cached_property` isn't really needed.
> I'd actually expect cached_property to have more overhead

Reply:
yea, makes sense. tbh, Parser could be a non-instanced class but oh well